### DISCUSSION TITLE
Raids suggestions. Bugs and Better experience.

### DISCUSSION THREAD

**Discussion by ZedNox** [chromiecraft/chromiecraft#3236] [Suggestions]
Till the suggestions come's here i was wondering.. Why are you guys are following the Blizzard path so hardly? Isn't it better to just make your own way so the game can acculy be more interesting since we are all casual's here?

You are going to relese bwl soon but still, tank's falling trought the texture in raggnaros and onxyia somtimes flys trought the ground as well. Just remove the weekly reset for raids at all so we can run's ad tell what is to fix. Just open up all the things so we have somthing to do or involve more people because as i can see we lost a lot of players that don't want to spend so much time and wasting time on

Morever, making nerf or buff just for delete them when the new content come out is pretty usless. Just let the game how it was to be, who care's if it's not challenging this bracet, we are going to sell all the stuffs when TBC will come out anyway.

You guys used to do a lot of better things before 60 bracet come out honeslty and I'am pretty sure to not be the only one thinking about that so all i'm on about was this, just wondering of the future of the server. Because as a player probably we gonna have Classic Wrath coming as well and i'm not suppose to say that but when that's probably come out we're gonna lose even more players. 

Just do the things easly and softly, i mean having a blizzlike experience is nice ok but we are no more in 2005 anyway.

**Comment by Annamaria-CC** [comment]
Hey ZedNox,

We are continuously working on updates. Some bugs require different devs then others. Some bugs take longer to fix. We are aware of this:
> tank's falling trought the texture in raggnaros and onxyia somtimes flys trought the ground as well.

The tank falling through the texture is actively being worked on, but it's a difficult one to fix. We hope to have that fixed soon. Onyxia flying through the floor is already fixed.

> involve more people

Feel free to help us out with bug triaging, bug fix testing or (recruitment of) developers. Feel free to contact me on Discord (temper 2272) if interested

To adress the rest of your suggestion. We have tuning in place to make content 
a) easier to test. If you can rush through each and every raid and not care about mechanics, then we will not get all the bug reports we need to make sure the content is needed
b) People like a bit of a challenge, without tuning lots of people would feel more bored more easily. 

We have no planned nerfs to the tuning as of now.

I hope this addressed your suggestion in a satisfactory way.
